microscopium/_util.py

```diff
@@ -1,6 +1,82 @@
 import numpy as np
 import numbers
 
+
+def generate_spiral(dim, direction, clockwise=False):
+    """Generate an dim * dim of sequental integers in a spiral pattern.
+
+    Many HCS systems use a spiral pattern to capture multiple fields in
+    for each well. This function generates a spiral pattern of integers
+    to be used by the montaging functions.
+
+    Parameters
+    ----------
+    dim : int
+        The dimension of the spiral. The function will return a dim * dim
+        array.
```

Comment:
Whoops yeah, [1, 1] :P (despite the troll proofs out there 1 does not indeed equal 0)